third_party/xla_client/BUILD

```diff
@@ -145,7 +145,8 @@ cc_library(
         "@com_google_absl//absl/numeric:int128",
         "@com_google_absl//absl/strings",
     ] + if_cuda_is_configured([
-        "@org_tensorflow//tensorflow/compiler/jit:xla_gpu_device",
+        "@org_tensorflow//tensorflow/compiler/xla/stream_executor/gpu:gpu_init",
```

Comment:
are these lines already in openxla?

Reply:
I know this line is already in openxla. how about line 149?

Reply:
yes, line 149 is in openxla too: https://github.com/openxla/xla/blob/48b3044abfd43479c43d83f2ef51215c28014806/xla/service/BUILD#L1211